src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -320,8 +320,8 @@ const WorkspaceDashboard = _.flow(
           return span({ key: tag, style: styles.tag }, [
             tag,
             Utils.canWrite(accessLevel) && h(Link, {
-              tooltip: 'Remove tag',
-              disabled: busy,
+              tooltip: Utils.editWorkspaceError(workspace) || 'Remove tag',
+              disabled: busy || Utils.editWorkspaceError(workspace),
```

Comment:
When the workspace is locked, the add tag menu is removed entirely, but the remove tag buttons are disabled. Should they both be removed or both be disabled?

Reply:
Resolved the inconsistency in 077a8e3fad0f6fc520c94124c15a3edf44ac12a3. The X button is now removed rather than disabled.

Reply:
👍  Since this isn't rendered when `Utils.editWorkspaceError(workspace)` returns a message, `Utils.editWorkspaceError(workspace)` could be removed from `tooltip` and `disabled`.